# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/nsIFontLoadCompleteCallback.idl
# Commit: 1b1a7d2f27c4
# Full Hash: 1b1a7d2f27c4428fb8a9ff42f0ba361ec0bb56e6
# Author: Cameron McCormack <cam@mcc.id.au>
# Date: 2019-06-07 09:47:19
# Regressor Bug: 1490792
# File Overlap Count: 1
# Description:
#   Bug 1490792 - Part 4: Perform OpenType sanitization OMT. r=jfkthame
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D33897
# ==============================================================================

diff -r f38b9eb31da7 -r 1b1a7d2f27c4 gfx/thebes/nsIFontLoadCompleteCallback.idl
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/thebes/nsIFontLoadCompleteCallback.idl	Thu Jun 06 19:08:52 2019 +0000
@@ -0,0 +1,13 @@
+/* -*- Mode: IDL; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*-
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "nsISupports.idl"
+
+[uuid(302dbf09-079b-4648-8a06-a0486c1749c0)]
+interface nsIFontLoadCompleteCallback : nsISupports
+{
+  void fontLoadComplete();
+};